docs/pages/_document.js

```diff
@@ -8,6 +8,7 @@ import createEmotionCache from 'docs/src/createEmotionCache';
 import { getMetaThemeColor } from 'docs/src/modules/brandingTheme';
 import GlobalStyles from '@mui/material/GlobalStyles';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
+import { getInitColorSchemeScript } from '@mui/joy/styles';
```

Comment:
@siriwatknp should `@mui/joy` be added to `docs/package.json` as dependency?
We use `_document` in MUI X and now `@mui/joy` is missing dependency.

Reply:
Nice catch, will update the docs and tag you in a PR.